### PR TITLE
Add new MacCloud v2 IPs and move them to a separate file

### PR DESCRIPTION
**Pull request by miketimofeev**
We've been provided with the new IP list yet it's really huge so it's better to keep it in a separate file.

### CODE REVIEW

**Comment by miketimofeev** [comment]
@andymckay could you please review and merge this PR?

File: ips.py [hunk 1]
```diff
@@ -34,13 +34,8 @@ def cidrs():
...
 
     # Add in the hard coded Mac IPs in CIDR format to join the downloaded IPs.
-    ips.add("199.7.166.9/32")
-    ips.add("199.7.166.10/31")
-    ips.add("199.19.85.25/32")
-    ips.add("199.19.85.26/31")
-    ips.add("208.83.5.226/31")
-    ips.add("208.83.5.228/31")
-    ips.add("208.83.5.230/32")
+    for line in open('macips.txt'):
```
Comment (andymckay) [inline ips.py]: Nit: let's use " not ' to keep consistent in the file
```suggestion
    for line in open("macips.txt"):
```

**Comment by andymckay** [comment]
I think you'll need to update `setup.py` so that when we build an sdist bundle it includes the ".txt" file, otherwise I believe it gets ignored:

```
 ~/c/actions-ips   add-more-maccloud-v2-ips ●  python3 setup.py sdist
running sdist
running egg_info
creating actions_ips.egg-info
writing actions_ips.egg-info/PKG-INFO
writing dependency_links to actions_ips.egg-info/dependency_links.txt
writing requirements to actions_ips.egg-info/requires.txt
writing top-level names to actions_ips.egg-info/top_level.txt
writing manifest file 'actions_ips.egg-info/SOURCES.txt'
reading manifest file 'actions_ips.egg-info/SOURCES.txt'
writing manifest file 'actions_ips.egg-info/SOURCES.txt'
running check
creating actions-ips-0.2
creating actions-ips-0.2/actions_ips.egg-info
copying files to actions-ips-0.2...
copying README.md -> actions-ips-0.2
copying setup.py -> actions-ips-0.2
copying actions_ips.egg-info/PKG-INFO -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/SOURCES.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/dependency_links.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/requires.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/top_level.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/zip-safe -> actions-ips-0.2/actions_ips.egg-info
Writing actions-ips-0.2/setup.cfg
creating dist
Creating tar archive
removing 'actions-ips-0.2' (and everything under it)
```

That would cause `pip install` of this repo to file because it wouldn't be able to find it. I think that's the case anyway.

**Comment by miketimofeev** [comment]
@andymckay add it as an additional data file, looks good now
```
thebestmbpever:fork-actions-ips mikhail.timofeev$ python3 setup.py sdist
running sdist
running egg_info
writing actions_ips.egg-info/PKG-INFO
writing dependency_links to actions_ips.egg-info/dependency_links.txt
writing requirements to actions_ips.egg-info/requires.txt
writing top-level names to actions_ips.egg-info/top_level.txt
writing manifest file 'actions_ips.egg-info/SOURCES.txt'
adding license file 'LICENSE' (matched pattern 'LICEN[CS]E*')
reading manifest file 'actions_ips.egg-info/SOURCES.txt'
writing manifest file 'actions_ips.egg-info/SOURCES.txt'
running check
creating actions-ips-0.2
creating actions-ips-0.2/actions_ips.egg-info
copying files to actions-ips-0.2...
copying LICENSE -> actions-ips-0.2
copying README.md -> actions-ips-0.2
copying macips.txt -> actions-ips-0.2
copying setup.py -> actions-ips-0.2
copying actions_ips.egg-info/PKG-INFO -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/SOURCES.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/dependency_links.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/requires.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/top_level.txt -> actions-ips-0.2/actions_ips.egg-info
copying actions_ips.egg-info/zip-safe -> actions-ips-0.2/actions_ips.egg-info
Writing actions-ips-0.2/setup.cfg
Creating tar archive
removing 'actions-ips-0.2' (and everything under it)
```
BTW, should I bump the version in the setup.py from 0.2 to 0.3?

**Comment by andymckay** [comment]
Great, thanks @miketimofeev, yep let's bump the version to 0.3 as well.

**Comment by miketimofeev** [comment]
@andymckay bumped, thanks!

**Comment by andymckay** [comment]
Thanks @miketimofeev!

**Pull request merged**